src/tests/validity/function_optional_argument_array_default.ab

```diff
@@ -0,0 +1,12 @@
+// Output
+// 1000
+
+fun sum_array(a : [Num] = [100,200,300,400]): Num {
```

Comment:
im pretty sure its supposed to be `a: [Num]`, not `a : [Num]`

Reply:
right now the tests are working in this way, maybe is just defining a array with numbers?